tests/pages/LiquidityPage.ts

```diff
@@ -24,6 +33,33 @@ export class LiquidityPage {
   static getTokenSearchField() {
     return cy.get('[id=token-search-input]')
   }
+  static getLiquidityPair() {
+    return cy.get('[data-testid=pair-card]').eq(2)
```

Comment:
This is not the best solution imo - you are not always sure if the pair you are looking for is gonna be 2. I think better way to do this would be to write some more specific data-testid or at least use .contains